### PR TITLE
Add uci

### PR DESCRIPTION
**Pull request by edchangy11**
- Added the uci data and two scripts that that in a dataset input and will reproduce the figure in the paper.
- Not sure if we want to keep the same file structure now for where the data is stored.

### CODE REVIEW

File: docs/figures/uci_classification.py [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+        likelihood=gpflow.likelihoods.Bernoulli(), inducing_variable=Z.copy()
+        , num_data=n_train, whiten=True)
+
+    gpflow.set_trainable(m_svgp_nat.q_mu, False)
+    gpflow.set_trainable(m_svgp_nat.q_sqrt, False)
+
+    models.append(m_svgp_nat)
+    names.append('svgp_nat')
+
+    m_cvi = t_SVGP(
```
Comment (thevincentadam) [inline docs/figures/uci_classification.py]: cvi

File: docs/figures/uci_classification.py [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    kernel=gpflow.kernels.Matern52(lengthscales=np.ones((1,x.shape[1]))*ell, variance=var),
+    likelihood=gpflow.likelihoods.Bernoulli(), inducing_variable=Z.copy()
+    , num_data=n_train)
+
+
+    # Turn off natural params
+    gpflow.set_trainable(m_cvi.lambda_1 , False)
```
Comment (thevincentadam) [inline docs/figures/uci_classification.py]: cvi

File: docs/figures/uci_classification.py [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+    gpflow.set_trainable(m_cvi.lambda_2_sqrt , False)
+
+    models.append(m_cvi)
```
Comment (thevincentadam) [inline docs/figures/uci_classification.py]: same

**Review by thevincentadam — CHANGES_REQUESTED** [review]

File: docs/figures/uci_classification.py (hunk 3 above)
Comment (edchangy11) [reply docs/figures/uci_classification.py]: changed

**Comment by thevincentadam** [comment]
Late reply sorry,
I still think it would be best to move the experiments in a dedicated folder, with the data and associated utils.

Definitely, I'd separate the utils that are data loading utilities from the actual method utils.
And my reasoning is that the notebooks are more proof of concept / illustrations than experiments here.

Let me know your thoughts

**Review by thevincentadam — APPROVED** [review]

**Pull request merged**